Code/BuildSystem/CMake/FindEzOpenXR.cmake

```diff
@@ -0,0 +1,88 @@
+
+# find the folder into which the OpenXR loader has been installed
+
+# early out, if this target has been created before
+if (TARGET ezOpenXR::Loader)
+	return()
+endif()
+
+set (EZ_OPENXR_DIR "EZ_OPENXR_DIR-NOTFOUND" CACHE PATH "Directory of OpenXR runtime installation")
+set (EZ_OPENXR_PREVIEW_DIR "" CACHE PATH "Directory of OpenXR preview include root")
+mark_as_advanced(FORCE EZ_OPENXR_DIR)
+mark_as_advanced(FORCE EZ_OPENXR_PREVIEW_DIR)
+
+ez_pull_architecture_vars()
```

Comment:
does not longer exist, use ez_pull_compiler_and_architecture_vars instead.

Reply:
Thanks for the tip, rebasing now.